lib/bibliothecary/parsers/maven.rb

```diff
@@ -75,6 +85,30 @@ def self.parse_ivy_report(file_contents)
         end.compact
       end
 
+      def self.parse_gradle_resolved(file_contents)
+        type = nil
+        file_contents.split("\n").map do |line|
+          type_match = GRADLE_TYPE_REGEX.match(line)
+          type = type_match.captures[0] if type_match
+
+          gradle_dep_match = GRADLE_DEP_REGEX.match(line)
+          next unless gradle_dep_match
+
+          split = gradle_dep_match.captures[0]
+
+          # org.springframework.boot:spring-boot-starter-web:2.1.0.M3 (*)
+          # Lines can end with (n) or (*) to indicate that something was not resolved (n) or resolved previously (*).
```

Comment:
Do we then end up with duplicates if something is resolved more than once?

Reply:
Yes. Should it dedupe?

Reply:
Oh, yeah - I hit this when I did it with the manual script. Dups actually break the frontend right now (throws a duplicate key exception about a Vue v-for).

Reply:
Yeah, pretty sure we want to dedupe in general since we're not storing the graph of why things get pulled in.